### ISSUE TITLE
Init feature

### ISSUE THREAD

**Issue by jawira** [phingofficial/phing#796]
It would be nice to have an "initialize" option for Phing.

I systematically look for this feature everytime I discover a new package.

This kind of feature is usual nowadays, to name a few:

  * Behat: `$ behat --init`
  * Composer: `$ php composer.phar init`
  * Phinx: `$ phinx init .`

So for Phing I'm wanting something like this:

```sh
$ vendor/bin/phing -init
```

```sh
$ vendor/bin/phing -i
```

Ideally this command would create two files at the root of the project: `build.xml` and `build.properties` (these are the default naming conventions).

The content of `build.xml`:

```xml
<?xml version="1.0" encoding="UTF-8" ?>

<project name="" description="" default="">
    
    <target name="" description="">
        
    </target>
    
</project>
```

The content of `build.properties`:

```ini
[section]
name=value
```

I have already worked on this, and I'm really expecting for your feedback.

**Comment by siad007** [comment]
Thanks for your PR!
Some small issues:

- do we really need a property file generated?
- if so, there is no support for sections by a property file but for ini files

I would suggest dropping the generation of the property file.

**Comment by siad007** [comment]
What do you think about a optional usage of a filename?
```
$ vendor/bin/phing -init "phing.xml"
```
It would also be nice to support
- a single file (phing.xml)
- with a relative path (test/phing.xml)
- or an absolute path (/path/to/phing.xml)
What do you think.

**Comment by Jawira** [comment]
Thanks a lot for your quick feedback!

About **generation of property file**: I added this because it's what I always use in all my projects, and I consider it's a good practice to keep all properties together in a same file, however, I agree that not all people need or want a property file.

Regarding sections, I know they are not supported by properties and are omitted, but I always use sections in my properties as categories:
```ini
[database]
db.user=user

[ftp]
ftp.server=

[tests]
dir.output=
```
But once again, I agree this couldn't be the case for all Phing's users. Therefore I will drop the generation of property file as you suggest, mainly for the sake of simplicity.

Concerning the **optional usage of a filename**, I'm not very keen about this. The main reason is that I want to keep the `-init` option really simple and straight forward. I always call my buildfile `build.xml` because this way I don't have to use the `-f` option and the command is shorter in terminal. Even if I'm not really against this idea I prefer keeping things simple.

**Comment by Jawira** [comment]
I updated my PR, I have taken into account your suggestions.

**Comment by mrook** [comment]
Closed because #797 was merged.
